### PR TITLE
Azure: monitor long running resources

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [x] enhancement
- [ ] documentation
- [ ] dependencies

## Description
Azure CloudSesei - report the long-running resources in Azure. 
- Currently sending slack notifications.

https://issues.redhat.com/browse/CLGOVNR-21

### CODE REVIEW

**Review by ebattat — COMMENTED** [review]
Did you add cost for each instance ?

**Comment by athiruma** [comment]
> Did you add cost for each instance ?

I will open a new PR to add cost reports. They requested that cost reports be included in the mail/Slack.

File: .gitignore [hunk 1]
```diff
@@ -217,3 +217,4 @@ empty_test_environment_variables.py
 /cloud_governance/main/.test_env
 /cloud_governance/policy/send_mail.py
 cloudsensei/.env.txt
+.vscode
```
Comment (krishvoor) [inline .gitignore:220]: Though harmless - do we need this :)

Comment (athiruma) [reply .gitignore:220]: We don't need to store vscode files in GitHub.

**Review by ebattat — APPROVED** [review]
/LGTM

File: .gitignore (hunk 1 above)
Comment (krishvoor) [reply .gitignore:220]: /lgtm

**Review by krishvoor — APPROVED** [review]
/lgtm

**Comment by athiruma** [comment]
> Can you not reuse this https://github.com/redhat-performance/cloud-governance/blob/main/cloudsensei/slack_operations.py -?

We need to create a bundle so that all files must be in one place.

**Comment by athiruma** [comment]
> > Can you not reuse this https://github.com/redhat-performance/cloud-governance/blob/main/cloudsensei/slack_operations.py -?
> 
> We need to create a bundle so that all files must be in one place.

I'll create a plan to make it easier. May be in new pr.

**Comment by krishvoor** [comment]
> > > Can you not reuse this https://github.com/redhat-performance/cloud-governance/blob/main/cloudsensei/slack_operations.py -?
> > 
> > 
> > We need to create a bundle so that all files must be in one place.
> 
> I'll create a plan to make it easier. May be in new pr.

All right, this PR is /lgtm. Please merge it!

**Review by krishvoor — APPROVED** [review]
/lgtm

**Pull request merged**